src/scheduling-service.ts

```diff
@@ -12,6 +12,40 @@ import {
   setSchedulingBooking,
 } from './services/schedulingHandlers'
 
+async function getPaymentConfiguration({
+  paymentReceiptUrl,
+  paymentFormUrl,
+  formSubmissionResult,
+  schedulingBooking,
+}: {
+  paymentReceiptUrl: string | undefined
+  paymentFormUrl: string | undefined
+  formSubmissionResult: FormSubmissionResult
+  schedulingBooking: SchedulingBooking
+}) {
+  //@ts-expect-error TODO update types
+  if (formSubmissionResult.preventPayment) {
```

Comment:
Doesn't this type exist in this code base?

Reply:
yes 🤦